webapp/src/pages/Settings.tsx

```diff
@@ -374,20 +375,20 @@ const Settings: React.FC<Props> = ({ onClose }) => {
     <StringField
       key={field}
       label={label}
-      value={String(value)}
+      value={value === null ? "" : String(value)}
```

Comment:
As I said earlier, that's as simple as
```suggestion
      value={value ?? ""}
```